samza-sql/src/main/java/org/apache/samza/sql/runner/SamzaSqlApplicationRunner.java

```diff
@@ -56,9 +57,20 @@
   public static final String RUNNER_CONFIG = "app.runner.class";
   public static final String CFG_FMT_SAMZA_STREAM_SYSTEM = "streams.%s.samza.system";
 
+  /**
+   * NOTE: This constructor is called from {@link ApplicationRunners} through reflection.
+   * Please refrain from updating the signature or removing this constructor unless the caller has changed the interface.
+   */
+  public SamzaSqlApplicationRunner(SamzaApplication app, Config config) {
+    this(app, false, config);
```

Comment:
@srinipunuru Does this assume that this will only be used in YARN? ApplicationRunners#getApplicationRunner will be used for both YARN and Standalone. What does setting localRunner == false do?

Let's also sync up offline about why we need a separate runner for SQL. Maybe we can reuse the existing runners.

Reply:
@prateekm Yes it will be used just in yarn. Regarding the need for separate runner we can chat about it. I can't remember who recommended this approach, But this was recommended approach to do config translation rather than use config rewriter. I had to rewrite the config rewriter as applicationRunner.